.github/workflows/docker-image.yml

```
@@ -9,15 +9,42 @@
       run: docker build . --file Dockerfile --tag vanneszias/useless-inventions:${{ github.sha }}
-
+      
     - name: Push the Docker image
       run: docker push vanneszias/useless-inventions:${{ github.sha }}
+
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Deploy to Azure Web App
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)